components/cliniko/actions/get-patient/get-patient.js

```diff
@@ -0,0 +1,21 @@
+const cliniko = require("../../cliniko.app.js");
+
+module.exports = {
+  name: "Get Patient",
+  key: "cliniko-get-patient",
+  description: "Get the details of a patient by `patientId`.",
```

Comment:
Markdown isn't rendered in the action description (we're working on that, though!). For now I'd remove the backticks.